system/lib/libc/musl/arch/emscripten/bits/alltypes.h

```diff
@@ -4,6 +4,14 @@
 
 #define __BYTE_ORDER __LITTLE_ENDIAN
 
+// Can't use __LONG_MAX__ here since musl's libs does pre-processor comparison
+// with 0x7fffffffL directly.
```

Comment:
comparing in the pre-processor should work fine even with `#define LONG_MAX __LONG_MAX__`? https://godbolt.org/z/zonffPn6o

Reply:
Hmm, in that case there must be some other reason.. I just assumed that was it.

Reply:
https://github.com/emscripten-core/emscripten/pull/20756